components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/console-list.js

```diff
@@ -86,8 +86,15 @@ define(['log', 'jquery', 'lodash', 'backbone', 'console'], function (log, $, _,
             showConsoleComponents: function () {
                 var self = this;
                 var consoleHeaderContainer = self._$parent_el;
-                $('#service-tabs-wrapper').css('height', '65%');
+                var serviceWrapper =  $('#service-tabs-wrapper');
+                serviceWrapper.css('height', '65%');
                 consoleHeaderContainer.removeClass('hide');
+                $("#service-tabs-wrapper").resizable({
```

Comment:
Shall we stick to single quotes in js files without using both.

Reply:
Fixed with #1690